modules/ROOT/pages/migrate-mule3-apimanager.adoc

```diff
@@ -34,7 +34,7 @@ If you applied custom policies to your Mule 3 API, follow xref:mule-runtime::mig
 [[determine-migration-approach]]
 == Determine the Migration Approach
 
-You migrate your Mule 3 application that is managed by API Manager using an existing API instance or by creating an instance. Before you decide the option, carefully consider the following migration factors to determine the approach that best suits your requirements:
+You migrate your Mule 3 application managed by API Manager either by using an existing API instance or by creating an instance. Before you decide the option to use, carefully consider the following migration factors to determine the approach that best suits your requirements:
```

Comment:
The first sentence repeats what has already been said a couple of times; can we omit it?
```suggestion
Before migrating to Mule 4, carefully consider the following factors to determine the approach that best suits your requirements:
```